.github/workflows/snapshot.yml

```diff
@@ -0,0 +1,109 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Copyright (C) 2020 by Tarek BOUCHKATI <tarek.bouchkati@gmail.com>
+
+on: push
+
+name: OpenOCD Snapshot
```

Comment:
This workflow runs an elaborate cross build for Windows under Linux, including build of dependencies. 

Could you also add a workflow that would run just a plain simple Linux build? (`./bootstrap` + `./configure` + `make` + `make install`) Thanks.

Reply:
Yes, I want to add 32-bit and 64-bit Linux builds. This one is here because it exists in mainline OpenOCD and we might as well use it.